README.md

```diff
@@ -1,26 +1,41 @@
 # ado-asana-sync
 
-[![Open in Visual Studio Code](https://img.shields.io/static/v1?logo=visualstudiocode&label=&message=Open%20in%20Visual%20Studio%20Code&labelColor=2c2c32&color=007acc&logoColor=007acc)](https://open.vscode.dev/danstis/ado-asana-sync)
+[![Test and Lint](https://github.com/danstis/ado-asana-sync/actions/workflows/build.yml/badge.svg)](https://github.com/danstis/ado-asana-sync/actions/workflows/build.yml)
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=danstis_ado-asana-sync&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=danstis_ado-asana-sync)
+[![Coverage](https://sonarcloud.io/api/project_badges/measure?project=danstis_ado-asana-sync&metric=coverage)](https://sonarcloud.io/summary/new_code?id=danstis_ado-asana-sync)
+
 [![Release](https://img.shields.io/github/release/danstis/ado-asana-sync.svg?style=flat-square)](https://github.com/danstis/ado-asana-sync/releases/latest)
+[![PyPI - Version](https://img.shields.io/pypi/v/ado-asana-sync)](https://pypi.org/project/ado-asana-sync/)
+
+[![Open in Visual Studio Code](https://img.shields.io/static/v1?logo=visualstudiocode&label=&message=Open%20in%20Visual%20Studio%20Code&labelColor=2c2c32&color=007acc&logoColor=007acc)](https://open.vscode.dev/danstis/ado-asana-sync)
+
+This project is a work in progress and is not yet ready for use. Breaking changes will occur as needed.
+
+## How to use
 
-## Code structure
+* Get the latest container image from the [Github Container Registry](https://github.com/danstis/ado-asana-sync/pkgs/container/ado-asana-sync).
+* Configure the environment variables with the relevant values:
+  * `ADO_PAT` - Your Personal Access Token for ADO to accesst the work items.
+  * `ADO_URL` - The full URL of your Azure DevOps instance.
+  * `ASANA_TOKEN` - Your Personal Access Token for Asana to access the work items.
+  * `ASANA_WORKSPACE_NAME` - Name of the Asana workspace to sync with.
 
-Projects should follow the folder structure from this standard project layout: [project-layout](https://github.com/golang-standards/project-layout)
+## Development
 
-## Commit message style
+### Commit message style
 
 This repo uses [Conventional Commits](https://www.conventionalcommits.org/) to ensure the build numbering is generated correctly
```

Comment:
**Suggestion:** It would be beneficial to include a 'Contributing' section in the README. This can guide potential contributors on how to get started with contributing to the project.
```suggestion
## Development

### Commit message style

This repo uses [Conventional Commits](https://www.conventionalcommits.org/) to ensure the build numbering is generated correctly

## Contributing

We welcome contributions! Please see our [Contributing Guide](CONTRIBUTING.md) for more details.
```